lib/presentation/widget/account_details/account_details_viewmodel.dart

```diff
@@ -51,6 +52,7 @@ class AccountDetailsViewModel extends BaseViewModel {
   final GetQuotaInteractor _getQuotaInteractor;
   final DeleteAllOfflineDocumentInteractor _deleteAllOfflineDocumentInteractor;
   final DeleteAllSharedSpaceOfflineInteractor _deleteAllSharedSpaceOfflineInteractor;
+  final LogoutOidcInteractor _logoutOidcInteractor = getIt<LogoutOidcInteractor>();
```

Comment:
U should inject it in the viewmodel to make it easier to manage later.

Reply:
IMO, for my way, we can not define much in `*_module` and not make too much think in constructor of viewModel. (you can see now, some view model have many interactors in constructor)

Reply:
IMO should reduce to one. If you want to fix it later, it will be very difficult to find. You should reconsider.

Reply:
ok, make it consistent with the way before